modules/alloydb/outputs.tf

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  #  _all_instances = merge(
+  #    { primary = google_alloydb_instance.primary },
+  #    google_alloydb_instance.replicas
+  #  )
+  _all_instances = {
+    primary   = google_alloydb_instance.primary
+    secondary = one(google_alloydb_instance.secondary)
+  }
+}
+
+output "id" {
```

Comment:
Should we just return a dictionary for each data? i.e. ids, ips, ..... and don't provide the single value? ...it's just a personale preference. I personally find it confusing and redundant

Reply:
I'm keeping id, ip to be aligned with other modules logic.